view/next-project/src/components/sponsoractivities/DetailModal.tsx

```diff
@@ -46,16 +46,56 @@ const DetailModal: FC<ModalProps> = (props) => {
           </p>
         </div>
         <div className='flex gap-3'>
-          <p>回収状況</p>
+          <p className='text-black-600'>回収状況</p>
           <p className='border-b border-primary-1'>
             {props.sponsorActivitiesViewItem.sponsorActivity.isDone ? '回収済み' : '未回収'}
           </p>
         </div>
         <div className='flex gap-3'>
-          <p>担当者名</p>
+          <p className='text-black-600'>担当者名</p>
           <p className='border-b border-primary-1'>{props.sponsorActivitiesViewItem.user.name}</p>
         </div>
       </div>
+      <div className='my-10 flex flex-wrap justify-center gap-8'>
+        <div className='flex gap-3'>
+          <p className='text-black-600'>オプション</p>
+          <p className='border-b border-primary-1'>
+            {props.sponsorActivitiesViewItem.sponsorActivity.feature}
+          </p>
+        </div>
+        <div className='flex gap-3'>
+          <p className='text-black-600'>交通費</p>
+          <p className='border-b border-primary-1'>
+            {props.sponsorActivitiesViewItem.sponsorActivity.expense}円
+          </p>
+        </div>
+      </div>
+      <p className='my-5 mx-auto w-fit text-xl text-black-600'>備考</p>
+      <table className='w-full table-fixed border-collapse'>
+        <thead>
+          <tr className='border border-x-white-0 border-b-primary-1 border-t-white-0 py-3'></tr>
+        </thead>
+        <tbody>
+          <tr className='border border-x-white-0 border-b-primary-1 border-t-white-0'>
+            <td className='py-3'>
+              <div>
+                <p className='border-primary-1'>
+                  {props.sponsorActivitiesViewItem.sponsorActivity.remark.length < 36 ? (
+                    <div className='text-center'>
+                      {props.sponsorActivitiesViewItem.sponsorActivity.remark === '' && (
+                        <div>なし</div>
+                      )}
+                      {props.sponsorActivitiesViewItem.sponsorActivity.remark}
+                    </div>
+                  ) : (
+                    <div>{props.sponsorActivitiesViewItem.sponsorActivity.remark}</div>
+                  )}
+                </p>
```

Comment:
ここはどういう処理を想定して36になって、こういう分岐にしたか聞いても良い？
備考がないなら「なし」と表示したいだけであれば、以下のようにかける！

```suggestion
                <p className='border-primary-1 text-center'>
                  {props.sponsorActivitiesViewItem.sponsorActivity.remark
                    ? props.sponsorActivitiesViewItem.sponsorActivity.remark
                    : 'なし'}
                </p>
```

Reply:
日本語文字列で36文字以下やと備考の欄で中央寄せにして見やすいかなって思った！36文字以上やと折り返すから左寄せになるようにした、まあなくても良いかも

Reply:
そういうことね！
それならclsx使ったほうが綺麗になるかも！

```suggestion
                <p className={clsx('border-primary-1', {
                  'text-center': props.sponsorActivitiesViewItem.sponsorActivity.remark.length < 36,
                })}>
                  {props.sponsorActivitiesViewItem.sponsorActivity.remark === '' && (
                    <div>なし</div>
                  )}
                  {props.sponsorActivitiesViewItem.sponsorActivity.remark}
                </p>
```

Reply:
修正した!
[clsx import](https://github.com/NUTFes/FinanSu/pull/565/commits/689677ac01533d40a93b1dadf2efd2c17fd50a4a)